activestorage/config/routes.rb

```diff
@@ -34,12 +34,12 @@
     if model.respond_to?(:signed_id)
       route_for(
         :rails_service_blob_proxy,
-        model.signed_id,
+        model.signed_id(expires_in: options.delete(:expires_in)),
```

Comment:
@zzak 

I think it would be more useful to make the default value configurable as follows, what do you think about that?

```ruby
model.signed_id(expires_in: options.delete(:expires_in) { ActiveStorage.rails_urls_expire_in }),
```

Reply:
Probably an application/global setting + an argument that overrides it makes sense. 🤔

Reply:
Thanks.
I tried the application settings.

70a9335